kimchi/src/circuits/gate.rs

```diff
@@ -273,6 +298,24 @@ impl<F: FftField> LookupInfo<F> {
     }
 }
 
+/// Specifies the relative position of gates and the fixed lookup table (if applicable) that a
```

Comment:
No biggy, but I always try to line break after punctuation and not in the middle of a sentence as it looks weird otherwise

Reply:
I'm already of the opinion that the comments in rust are too long (they're a nuisance to read on a phone, for example), I'd prefer not to make this line even longer.

Reply:
These are really meant to be read on the generated docs though